src/region.proto

```diff
@@ -7,6 +7,7 @@ enum region {
   EU868 = 1;
   EU433 = 2;
   CN470 = 3;
+  // CN779 is deprecated.  Please remove.
```

Comment:
leave the deprecated mark in there but you can't just remove the entry

Reply:
In theory removing enum entries is supported.  As long as the physical number is not re-used.
https://developers.google.com/protocol-buffers/docs/proto

Reply:
Its not only about whether its supported by protobuf but about whether it gets used in some critical path in core/miner (this is also true for any other field in other protos btw). That said I'm pretty sure CN779 isn't used anywhere but having it here also doesn't hurt.

If at some point we are to change the region lookup mechanism we can be more thorough.

Reply:
In addition changing existing names will cause other tools to break (like gateway-rs)

Reply:
OK.  I removed the 'remove' suggestion.